tests/e2e/specs/shopper/filter-products-by-attribute.test.ts

```diff
@@ -227,4 +229,121 @@ describe( `${ block.name } Block`, () => {
 			);
 		} );
 	} );
+
+	/*
+	 * @todo Remove this logic when WordPress 6.1 is released. This is a temporary fix because WP 6.0 doesn't ship the necessary changes to make it work properly the Product Query block.
+	 */
```

Comment:
I'm not sure to understand this comment, does it refer to the entire `describe()` clause? If so, why it wasn't necessary in the _Filter by Price_ block tests?

Reply:
I forgot to remove this comment! I worked on this PR before that WordPress 6.1 was released. Now, it isn't necessary anymore!